AppInspector/rules/default/cryptography/hash_algorithm.json

```diff
@@ -105,30 +83,23 @@
     "name": "Cryptography: Hash Algorithm (General)",
     "id": "AI009200",
     "description": "Cryptography: Hash Algorithm (General)",
-    "tags": [
-      "Cryptography.HashAlgorithm.General"
-    ],
+    "tags":[ "Cryptography.HashAlgorithm.General" ],
     "severity": "moderate",
     "patterns": [
       {
         "pattern": "hash",
         "type": "string",
-        "scopes": [
-          "code",
-          "comment"
-        ],
+        "scopes": [ "code", "comment" ],
         "modifiers": [ "i" ],
         "confidence": "medium"
       }
     ],
     "conditions": [
       {
         "pattern": {
-          "pattern": "(NS)*Hashset|Hashtable|Hashmap",
```

Comment:
I'm not sure this is doing what we want it to -- I think the point was to not find `hash` when it's near `hashmap`, which is probably too unpredictable to have value. Maybe we just remove the conditional section here and keep the confidence as low for 'hash'.